### PR TITLE
feat(upshot): Build Upshot Adapter

### PR DESCRIPTION
**Pull request by 1marcghannam**

### CODE REVIEW

File: packages/sources/upshot/src/endpoint/floor-price.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { ResponseSchema } from '../lib/types'
+import { AssetId, assetIdToAddress } from '../lib/const'
+import { join } from 'path'
+
+export const supportedEndpoints = ['floor-price']
+
+export const endpointResultPaths = {}
```
Comment (jnapprogs) [inline packages/sources/upshot/src/endpoint/floor-price.ts:9]: Is it safe to remove this variable?

**Review by jnapprogs — CHANGES_REQUESTED** [review]
Had a quick question, but looks good 👍

File: packages/sources/upshot/src/endpoint/floor-price.ts (hunk 1 above)
Comment (vnavascues) [reply packages/sources/upshot/src/endpoint/floor-price.ts:9]: Good question. It is safe to delete it but also Chainlink leaves it empty, so let's keep it empty.

**Review by vnavascues — APPROVED** [review]
LGTM!

**Pull request merged**